### PR TITLE
feat(nova): added redirect to feedback page after call end

### PR DESCRIPTION
**Pull request by mostafakamar2308**

### CODE REVIEW

File: apps/nova/src/pages/Callfeedback.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { Button, ButtonType } from "@litespace/luna";
+import React from "react";
+import { Link } from "react-router-dom";
+
+const CallFeedback: React.FC = () => {
+  return (
+    <div className="flex items-center justify-center">
+      <Link to={"/"}>
```
Comment (neuodev) [inline apps/nova/src/pages/Callfeedback.tsx:8]:
```suggestion
      <Link to={Route.Home}>
```

File: apps/nova/src/pages/Call.tsx [hunk 2]
```diff
@@ -144,7 +147,10 @@ const Call: React.FC = () => {
   }, [callId, startRecording, userMediaStream]);
 
   const onLeaveCall = useCallback(() => {
+    // TODO: This didn't stop the camera and needs further investigations
+    stop();
     peer.destroy();
+    id && navigate(Route.CallFeedback.replace(":id", id));
```
Comment (neuodev) [inline apps/nova/src/pages/Call.tsx:153]:
```suggestion
   if (id) navigate(Route.CallFeedback.replace(":id", id));
```

File: apps/nova/src/pages/Call.tsx [hunk 3]
```diff
@@ -144,7 +147,10 @@ const Call: React.FC = () => {
   }, [callId, startRecording, userMediaStream]);
 
   const onLeaveCall = useCallback(() => {
+    // TODO: This didn't stop the camera and needs further investigations
```
Comment (neuodev) [inline apps/nova/src/pages/Call.tsx:150]: Nice 🚀 !!

**Review by neuodev — CHANGES_REQUESTED** [review]

**Comment by mostafakamar2308** [comment]
Change requests implemented 🚀

**Comment by neuodev** [comment]
> Change requests implemented 🚀

Thank you! 

Please click "Re-request review" once you are done with your changes. 

image

**Review by neuodev — APPROVED** [review]

**Comment by mostafakamar2308** [comment]
@neuodev  
Do we need to migrate this to master directly? As I see, It's not related to migrate-queries.

**Comment by neuodev** [comment]
Another solution, is to rebase you work on top of the latest master and drop unrelated commits.

**Comment by neuodev** [comment]
One last comment is the commit/PR title should start by lowercase. 

```diff
- feat(nova): Added redirect to feedback page after call end
+ feat(nova): added redirect to feedback page after call end
```

**Comment by neuodev** [comment]
> One last comment is the commit/PR title should start by lowercase.
> 
> ```diff
> - feat(nova): Added redirect to feedback page after call end
> + feat(nova): added redirect to feedback page after call end
> ```

@mostafakamar2308 please update the PR title and the commit message.

**Review by neuodev — APPROVED** [review]

**Comment by neuodev** [comment]
@mostafakamar2308 please rebase this branch on top of `master` as I cannot find a reason why it is based on `mk/migrate-queries`

**Comment by neuodev** [comment]
Please rebase you merge request and resolve the conflicts @mostafakamar2308!

**Pull request closed without merging**